src/sentry/static/sentry/less/layout.less

```diff
@@ -37,13 +38,6 @@ body {
     &.collapsed {
       padding-left: @sidebar-collapsed-width;
     }
-
-    /* This is for organization level views e.g. "Assigned to me" */
-    .organization-home {
```

Comment:
There may be a few server-rendered views that use this class btw (I'm not sure though).

Reply:
It looks like `organization-home` is present in sentry/bases/organization.html, but it's not being extended anywhere as far as I can tell. Probably safe to kill.